rules/integrations/cloud_defend/execution_interactive_shell_spawned_from_inside_a_container.toml

```diff
@@ -0,0 +1,57 @@
+[metadata]
+creation_date = "2023/04/26"
+integration = ["cloud_defend"]
+maturity = "production"
+min_stack_comments = "New Integration: Cloud Defend"
+min_stack_version = "8.8.0"
+updated_date = "2023/05/04"
+
+[rule]
+author = ["Elastic"]
+description = "This rule detects when an interactive shell is spawned inside a running container. This could indicate a potential container breakout attempt or an attacker's attempt to gain unauthorized access to the underlying host."
+false_positives = ["""
+  Legitimate users and processes, such as system administration tools, may utilize shell utilities inside a container resulting in false positives.
+  """
+]
+from = "now-6m"
+index = ["logs-cloud_defend*"]
+interval = "5m"
+language = "eql"
+license = "Elastic License v2"
+name = "Suspicious Interactive Shell Spawned From Inside A Container"
+tags = ["Elastic", "Host", "Linux", "Container", "Threat Detection", "Execution"]
+risk_score = 73
+rule_id = "8d3d0794-c776-476b-8674-ee2e685f6470"
+severity = "high"
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = """
+process where container.id: "*" and
+event.type== "start" and 
+/*D4C consolidates closely spawned event.actions, this excludes end actions to only capture ongoing processes*/
+event.action in ("fork", "exec") and not event.action== "end"
```

Comment:
I believe this line could be made simpler by just changing it to. This would capture any event action that's not end.
```sql
event.action != "end"